docs/changelog/89599.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 89599
+summary: "Fleet: Add new mappings for .fleet-agents `last_checkin_message` and components\
+  \ fields"
```

Comment:
Is this line break intended? Or should it be all in one line?
```suggestion
summary: "Fleet: Add new mappings for .fleet-agents `last_checkin_message` and components fields"
```

Reply:
this file was automagically creating by the existing tooling, so I assume it knows what it's doing

Reply:
good point, I'd assume the same